indexer/indexer_tests/osm_editor_test.cpp

```diff
@@ -167,7 +167,7 @@ void EditorTest::GetFeatureTypeInfoTest()
   auto const mwmId = ConstructTestMwm([](TestMwmBuilder & builder)
   {
     TestCafe cafe(m2::PointD(1.0, 1.0), "London Cafe", "en");
-    builder.Add(cafe);
+    builder.Add(cafe);    
```

Comment:
We probably don't but I'm used to clang-format removing them (which it didn't here).